uwblueprint/community-fridge-kw
# style: admin user management

## Brief description. What is this change? 

### [Admin User Management FE Fixes](https://www.notion.so/uwblueprintexecs/Admin-User-Management-FE-Fixes-f7fe69883075468988bd2bac66aa995d)



## Implementation description. How did you make this change?
* Built upon previous user management page



## Steps to test
1. Go to user management page. Approve a volunteer and ensure a correct toast appears.
2. Delete a volunteer and ensure a correct toast appears.
3. For volunteers, all companies should appear as "-"
4. For volunteers that are approved, a checkmark icon should appear.
5. For donors, the approval status should be "-"



## Checklist
